src/repeat.js

```diff
@@ -120,6 +120,9 @@ export class Repeat {
 
     let items = this.items;
     this.strategy = this.strategyLocator.getStrategy(items);
+    if (!this.strategy) {
+      return;
```

Comment:
I think throwing instead of returning would be good here. It would keep the behavior the same while improving the developer experience by giving clear information why things are broken.  I think you could even include `this.sourceExpression.toString()` in the error message so the developer will know which binding expression is returning a non-repeatable value.